### PR TITLE
allow for gas-water + brine

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by goncalvesmachadoc** [comment]
The only failure is the new test itself since it has nothing to compare with. Can it be reviewed and merged?

**Comment by totto82** [comment]
Thanks. I will update ref data and merge.

**Comment by totto82** [comment]
jenkins build this update_data please

**Comment by akva2** [comment]
duplicate case name is no go.

File: opm/simulators/flow/Main.hpp [hunk 1]
```diff
@@ -639,10 +640,18 @@ class Main
...
+            if (phases.active(Phase::OIL)){ // oil water brine case
+                flowEbosOilWaterBrineSetDeck(
+                    setupTime_, deck_, eclipseState_, schedule_, summaryConfig_);
+                return flowEbosOilWaterBrineMain(argc_, argv_, outputCout_, outputFiles_);
+            }
+            if (phases.active(Phase::GAS)){ // gas water brine case
+                flowEbosGasWaterBrineSetDeck(
+                    setupTime_, deck_, eclipseState_, schedule_, summaryConfig_);
+                return flowEbosGasWaterBrineMain(argc_, argv_, outputCout_, outputFiles_);
+            }
```
Comment (bska) [inline opm/simulators/flow/Main.hpp:654]: There should be
```C++
return EXIT_FAILURE
```
possibly accompanied by some logging, here to cover the **unlikely** case that neither oil nor gas are active.

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=711 please

**Review by bska — APPROVED** [review]
The new reference solutions have been installed on the CI system.  I'll merge this now.  Thanks a lot!

**Pull request merged**